Formula/aubio.rb

```diff
@@ -23,33 +23,55 @@ class Aubio < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "numpy"
-  depends_on "python@3.10"
+  depends_on "python@3.11"
 
   on_linux do
     depends_on "libsndfile"
   end
 
-  resource "aiff" do
+  resource "homebrew-aiff" do
     url "https://archive.org/download/TestAifAiffFile/02DayIsDone.aif"
     sha256 "bca81e8d13f3f6526cd54110ec1196afd5bda6c93b16a7ba5023e474901e050d"
   end
 
+  # Fix build with Python 3.11 using Fedora patch. Failure is due to old waf 2.0.14.
+  # Remove on next release as HEAD has newer waf.
+  patch do
+    url "https://src.fedoraproject.org/rpms/aubio/raw/29fb7e383b5465f4704b1cdc7db27df716e1b45c/f/aubio-python39.patch"
+    sha256 "2f9cb8913b1c4840588df2f437f702c329b4de4e46eff4dcf68aff4b5024a358"
+  end
```

Comment:
Using Fedora patch because:
- HEAD doesn't include waf code but has script to fetch waf instead, which is probably run while generating tarball.
- waf fix (https://gitlab.com/ita1024/waf/-/commit/68997828c850ce7fb30b73b4adfde35053e539d1) also modifies  other files so can't apply directly.